taipy/gui/_gui_section.py

```diff
@@ -48,14 +48,14 @@ def _update(self, config_as_dict: t.Dict[str, t.Any], default_section=None):
 
     @staticmethod
     def _configure(**properties) -> "_GuiSection":
-        """NOT DOCUMENTED
-        Configure the Graphical User Interface.
+        """Configure the Graphical User Interface.
```

Comment:
Why document that? How requested it?

Reply:
This is the method dynamically injected into the config. The usage is:
```
from taipy import Config

Config.configure_gui(...)
```

By the way, a user asked how to do it in 3.1 since it was not documented.

If you prefer not to promote this way of configuring the GUI, I can remove it without hesitation.

Reply:
No no... I don't really like this thing but go ahead. That's homogeneous at least.
Thx